packages/location-management/src/components/LocationUnitDetail/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,226 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`location-management/src/components/LocationUnitDetail renders without crashing 1`] = `
```

Comment:
Is it possible to reduce this snapshot?

Reply:
Done i reduced it to min by using shallow but about the rest i think we should keep them as a part of snaphot to make sure everydetail is working fine